frontend/src/component/personalDashboard/ProjectSetupComplete.tsx

```diff
@@ -40,6 +40,91 @@ export const ProjectSetupComplete: FC<{ project: string }> = ({ project }) => {
                 </Link>{' '}
                 to work with existing flags
             </Typography>
+        </>
+    );
+};
+
+type HeathTrend = 'consistent' | 'improved' | 'declined' | 'unknown';
+
+const determineProjectHealthTrend = (
+    insights: PersonalDashboardProjectDetailsSchemaInsights,
+) => {
+    let trend: HeathTrend = 'unknown';
+    if (
+        insights.avgHealthCurrentWindow !== null &&
+        insights.avgHealthPastWindow !== null
+    ) {
+        if (insights.avgHealthCurrentWindow > insights.avgHealthPastWindow) {
+            trend = 'improved';
+        } else if (
+            insights.avgHealthCurrentWindow < insights.avgHealthPastWindow
+        ) {
+            trend = 'declined';
+        } else if (
+            insights.avgHealthPastWindow === insights.avgHealthCurrentWindow
+        ) {
+            trend = 'consistent';
+        }
+    }
+    return trend;
```

Comment:
we *could* do an early return here by checking if they're both null and just return the string directly. would decrease nesting by one level and remove the mutable variable. Your choice, though

Reply:
100%. Fixing